content/docs/contributing/to-docs/templates/component-reference-doc.md

```diff
@@ -0,0 +1,428 @@
+
+See `template-component-reference.md`:
+```md
+
+## <Component feature>
+
+<!-- (Optional) Explain specific features of your component. Repeat this section for each additional topic, as needed. -->
+
+```
+
+### Request Bus / Notification Bus (Optional)
+
+The request and notification bus sections contain the functions that users can call in their project's logic and scripts. 
+
+In the request or notification bus table, list the functions in rows and fill in the following columns:
+
+* **Request/Notification Name**: The name of the function., formatted in code-style.
+* **Description**: Describes what the function does. Be clear and concise, and explain any technical terminology. 
+* **Parameter**: What parameters this function accepts, if any. If none, write "None".
+* **Return**: What this function returns. 
+* **Scriptable**: Whether or not this function is scriptable. Write "Yes" or "No". 
+
+**Specifications**:
+
+* Add a sentence specifying the name of the EBus interface and which components the EBus interface communicates with. (Some groups of components share the same EBus.)
+
+* Separate request functions into the "Request Bus" section and notification functions into the "Notification Bus" section. 
+
+* Place the sections at the end of the page. 
+
+**Examples**:
+
+* [Terrain Physics Heightfield Collider component](/docs/user-guide/components/reference/terrain/terrain-physics-collider/)
+* [Vegetation Altitude Filter component](/docs/user-guide/components/reference/vegetation-filters/vegetation-altitude-filter/)
+* [Shape Reference component](/docs/user-guide/components/reference/shape/shape-reference/)
+
+
+See `template-component-reference.md`:
+```md
+
+## <EBus interface name>RequestBus
+
+<!-- (Optional) Example: "Use the following request functions with the <Request EBus name> EBus interface to communicate with <components> in your game." -->
+
+| Request Name | Description | Paramter | Return | Scriptable |
+| - | - | - | - | - |
+|   |   |   |   |   |
+|   |   |   |   |   |
+|   |   |   |   |   |
+
+## <EBus interface name>NotificationBus
+
+<!-- (Optional) Example: "Use the following notification functions with the <Notification EBus name> EBus interface to communicate with <components> in your game." -->
+
+| Request Name | Description | Paramter | Return | Scriptable |
+| - | - | - | - | - |
+|   |   |   |   |   |
+|   |   |   |   |   |
+|   |   |   |   |   |
+
+```
+
+
+## Where does the component reference document live?
+
+Component reference documentation live in the folder: `o3de.org/docs/user-guide/components/reference/`. They are organized into subfolders that group the components by type. Your component reference document may belong to an existing group, or a new group completely. 
+
+Feel free to reach out to D&C SIG for assistance. 
```

Comment:
```suggestion
Feel free to reach out to the [O3DE Documentation and Community Special Interest Group](https://github.com/o3de/sig-docs-community) for assistance. 
```

Reply:
Combined your and willihay's suggestion.
It now says: "Feel free to reach out to sig-docs-community on [Discord](https://discord.com/invite/o3de) or [GitHub](https://github.com/o3de/sig-docs-community) for assistance."